testapps/php56_e2e/composer.json

```diff
@@ -1,6 +1,15 @@
 {
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/stanley-cheung/Protobuf-PHP"
```

Comment:
We tried, but the upstream repo has not responded in a year: https://github.com/drslump/Protobuf-PHP/pull/48

But because he has submited his package to Packagist, we can't submit our fork unless we change the repo name.

That was why I have to resort to this `composer.json` syntax to use his official package on Packagist in the `require` section but substitute the actual content with our fork.